client/src/FileManager.module.css

```diff
@@ -1,37 +1,57 @@
 .FileManagerWrapper {
     width: 100%;
-    height: calc( 100vh - 108px - 60px - 24px - 56px - 2px - 10px );
+    height: calc(100vh - 108px - 60px - 24px - 56px - 2px - 10px);
 }
+
 .FileManager {
     width: 100%;
     height: 100%;
     border: none;
 }
 
-.ButtonGroup :first-child {
-    border-left-color: transparent;
+.FileToolBar {
+    height: 87px;
```

Comment:
There are a few issues that arise if the user changes their font size. Here, it would be better to use `rem` rather than `px` so the toolbar height would scale with a user's font size

There are also a few toolbar buttons where the icon goes outside the button in this situation (i.e. the new file, ne directory, rename buttons)

extreme example:
![image](https://github.com/openflighthpc/flight-file-manager/assets/102584263/89a7127e-f912-4cce-bcdd-451814be60fe)